testing/test_parseopt.py

```diff
@@ -288,8 +288,16 @@ def test_multiple_metavar_help(self, parser: parseopt.Parser) -> None:
 
 
 def test_argcomplete(testdir, monkeypatch) -> None:
-    if not shutil.which("bash"):
-        pytest.skip("bash not available")
+    try:
+        bash_version = subprocess.run(
+            ["bash", "--version"], capture_output=True, universal_newlines=True
+        ).stdout
+    except Exception:
```

Comment:
`Exception` seems quite broad here, maybe just `FileNotFoundError` or `OSError`?

Reply:
Right, I think `OSError` should be good for this.